transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DefaultInvocationSerializer.java

```diff
@@ -329,6 +338,112 @@ private String nameForClass(Class<?> clazz) {
     }
   }
 
+  static final class LocalDateTimeTypeAdapter extends TypeAdapter<LocalDateTime> {
+
+    @Override
+    public void write(JsonWriter out, LocalDateTime value) throws IOException {
+      out.value(value.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
```

Comment:
Should these have the null checks used in `UtcTypeAdapter`?

Reply:
nor required, added a test for null